packages/carbon-react/src/index.js

```diff
@@ -0,0 +1,201 @@
+  TableToolbarAction,
+  TableToolbarContent,
+  TableToolbarSearch,
+  TableToolbarMenu,
+  DatePicker,
+  DatePickerInput,
+  Dropdown,
+  ErrorBoundary,
+  ErrorBoundaryContext,
+  FileUploader,
+  Filename,
+  FileUploaderButton,
+  FileUploaderDropContainer,
+  FileUploaderItem,
+  Form,
+  FluidForm,
+  FormGroup,
+  FormItem,
+  FormLabel,
+  Grid,
+  Row,
+  Column,
+  InlineLoading,
+  Link,
+  ListItem,
+  Loading,
+  Modal,
+  MultiSelect,
+  ToastNotification,
+  InlineNotification,
+  NotificationActionButton,
+  NotificationButton,
+  NotificationTextDetails,
+  NumberInput,
+  OrderedList,
+  OverflowMenu,
+  OverflowMenuItem,
+  Pagination,
+  PaginationNav,
+  PrimaryButton,
+  ProgressIndicator,
+  ProgressStep,
+  RadioButton,
+  RadioButtonGroup,
+  Search,
+  SearchFilterButton,
+  SearchLayoutButton,
+  SecondaryButton,
+  Select,
+  SelectItem,
+  SelectItemGroup,
+  Switch,
+  Slider,
+  StructuredListWrapper,
+  StructuredListHead,
+  StructuredListBody,
+  StructuredListRow,
+  StructuredListInput,
+  StructuredListCell,
+  Tab,
+  TabContent,
+  Tabs,
+  Tag,
+  TextArea,
+  TextInput,
+  Tile,
+  ClickableTile,
+  SelectableTile,
+  ExpandableTile,
+  TileAboveTheFoldContent,
+  TileBelowTheFoldContent,
+  RadioTile,
+  TileGroup,
+  TimePicker,
+  TimePickerSelect,
+  Toggle,
+  ToggleSmall,
+  Toolbar,
+  ToolbarItem,
+  ToolbarTitle,
+  ToolbarOption,
+  ToolbarDivider,
+  ToolbarSearch,
+  Tooltip,
+  TooltipDefinition,
+  TooltipIcon,
+  UnorderedList,
+  SkeletonText,
+  SkeletonPlaceholder,
+  DataTableSkeleton,
+  AccordionSkeleton,
+  BreadcrumbSkeleton,
+  ButtonSkeleton,
+  CheckboxSkeleton,
+  CodeSnippetSkeleton,
+  DropdownSkeleton,
+  FileUploaderSkeleton,
+  NumberInputSkeleton,
+  PaginationSkeleton,
+  ProgressIndicatorSkeleton,
+  RadioButtonSkeleton,
+  SearchSkeleton,
+  SelectSkeleton,
+  SliderSkeleton,
+  StructuredListSkeleton,
+  TabsSkeleton,
+  TagSkeleton,
+  TextAreaSkeleton,
+  TextInputSkeleton,
+  ToggleSkeleton,
+  ToggleSmallSkeleton,
+  IconSkeleton,
+  DatePickerSkeleton,
+  Content,
+  Header,
+  HeaderContainer,
+  HeaderGlobalAction,
+  HeaderGlobalBar,
+  HeaderMenu,
+  HeaderMenuButton,
+  HeaderMenuItem,
+  HeaderName,
+  HeaderNavigation,
+  HeaderPanel,
+  HeaderSideNavItems,
+  Switcher,
+  SwitcherItem,
+  SwitcherDivider,
+  SkipToContent,
+  SideNav,
+  SideNavDetails,
+  SideNavDivider,
+  SideNavFooter,
+  SideNavHeader,
+  SideNavIcon,
+  SideNavItem,
+  SideNavItems,
+  SideNavLink,
+  SideNavLinkText,
+  SideNavMenu,
+  SideNavMenuItem,
+  SideNavSwitcher,
+  // Experimental
+  unstable_PageSelector,
+  unstable_Pagination,
+  unstable_TreeView,
+  unstable_TreeNode,
+  unstable_ContextMenu,
+  unstable_ContextMenuDivider,
+  unstable_ContextMenuGroup,
+  unstable_ContextMenuItem,
+  unstable_ContextMenuRadioGroup,
+  unstable_ContextMenuSelectableItem,
+  unstable_useContextMenu,
+  unstable_Heading,
+  unstable_Section,
+} from 'carbon-components-react';
```

Comment:
For option 3, is this really just using a relative path in these imports? Possible because we have a monorepo? That seams the cleanest until yarn releases its fix?

Reply:
@mattrosno basically we could get away with this by using a relative path and then Rollup could "inline it" in the resulting bundle, I think. Would have to prototype it to find out